webapi/src/main/twirl/queries/sparql/v1/searchExtended.scala.txt

```diff
@@ -27,14 +27,16 @@
  * Performs an extended search. The number of rows returned per matching resource is the product of the number of
  * values that each search criterion matched in the resource.
  *
+ * If the triplestore type is GraphDB, the generated SPARQL is optimised by taking advantage of forward-chaining
+ * RDFS inference.
+ *
```

Comment:
We should put that information in `/searchExtended.scala.txt `, `searchFulltext.scala.txt `, and `getResourceSearchResult.scala.txt `